akka-http-core/src/test/scala/akka/http/impl/engine/client/HostConnectionPoolSpec.scala

```diff
@@ -173,7 +173,7 @@ class HostConnectionPoolSpec extends AkkaSpec(
       "time out and reconnect when a request is not handled in time" in pending
       "time out when connection cannot be established" in pending
       "fail a request if the request entity fails" in new SetupWithServerProbes {
-        val reqBytesOut = pushChunkedRequest()
+        val reqBytesOut = pushChunkedRequest(numRetries = 0)
```

Comment:
was this needed?

Reply:
The test succeeded before, but would keep retrying after this particular test had already finished, making the logging for the *next* test very confusing to read :).